demo/kserve/metrics.md

```diff
@@ -1,26 +1,29 @@
-# Access Metrics
-Before proceding, please ensure you have [installed](/docs/README.md) all pre-requisites, required operators and components
+# Accessing Metrics
 
-## Setup
 
-To access the metrics, let's enable and configure the User Workload Monitoring.
-~~~
-oc apply -f uwm-cm-enable.yaml -n $TEST_NS
-oc apply -f uwm-cm-conf.yaml-n $TEST_NS
-~~~
-If desired, the retention time can be changed in `uwm-cm-conf.yaml`.
+**Prerequisites**
 
-## Deploy a model
+Before you begin, complete the prerequisites and install the required operators and components as described in the [Caikit-TGIS-Serving readme file](/docs/README.md).
 
-Please deploy a model following instructions [here](/docs/deploy-remove.md).
+**Procedure**
 
-## Accessing Caikit, TGIS and Istio metrics
+1. Enable and configure User Workload Monitoring.
 
-Navigate to Openshift Console --> Observe --> Metrics.
+   ~~~
+   oc apply -f uwm-cm-enable.yaml -n $TEST_NS
```

Comment:
Actually here, could we mention here that they have to be in `/demo/kserve/` directory and update the commands to:
```
oc apply -f custom-manifests/metrics/uwm-cm-enable.yaml -n $TEST_NS
oc apply -f custom-manifests/metrics/uwm-cm-conf.yaml -n $TEST_NS
```